src/components/NavBar/NavBar.tsx

```diff
@@ -3,17 +3,19 @@ import { NavLink } from "react-router-dom"
 import "./NavBar.css"
 
 export const NavBar: React.FC = () => {
+  const navActiveStyle = {
```

Comment:
Ohhh nice job breaking this into it's own variable to be reused for each navlink. So much cleaner!